### PR TITLE
Fix source maps

### PR DESCRIPTION
**Pull request by 7h3kk1d**
image

Sets up source maps in dev so you can get actual line numbers/debugging in stack traces

### CODE REVIEW

File: src/haz3lcore/dune [hunk 1]
```diff
@@ -4,8 +4,7 @@
  (name haz3lcore)
  (libraries util re sexplib unionFind uuidm yojson core)
  (js_of_ocaml
-  (flags
-   (:include js-of-ocaml-flags-%{profile})))
+  (flags :standard --debuginfo --noinline --dynlink --linkall --sourcemap))
```
Comment (7h3kk1d) [inline src/haz3lcore/dune]: Try going back to the profiles

Comment (7h3kk1d) [reply src/haz3lcore/dune]: The rules didn't work so I set it on the env

**Review by cyrus- — CHANGES_REQUESTED** [review]
Seems to work for me! Haven't robustly tested but breakpoints and exceptions seem to correctly work, which is huge.

There is a section on Source Maps in `README.md` in the root directory that needs to be updated. 

Can you also include these flags in the `dune` files in the other top-level libraries (`pretty`, `haz3lschool`)?

**Review by disconcision — APPROVED** [review]
works for me!

**Review by cyrus- — APPROVED** [review]

**Pull request merged**